ui/raidboss/data/triggers/o8n.js

```diff
@@ -6,110 +6,401 @@
+      regexFr: /14:291D:Kefka starts using Méga Foudre Fourmillante/,
       suppressSeconds: 1,
-      infoText: 'True Thunder',
-      tts: 'true',
+      infoText: {
+        en: 'True Thunder',
+        fr: 'Vraie foudre',
+        de: 'Wahrer Blitz',
+      },
+      tts: {
+        en: 'True',
+        fr: 'Vrai',
+        de: 'Wahr',
+      },
     },
     {
       id: 'O8N Thrumming Thunder Fake',
       regex: /14:291B:Kefka starts using Thrumming Thunder/,
+      regexDe: /14:291B:Kefka starts using Brachiales Blitzga/,
+      regexFr: /14:291B:Kefka starts using Méga Foudre Fourmillante/,
       suppressSeconds: 1,
-      infoText: 'Fake Thunder',
-      tts: 'fake',
+      infoText: {
+        en: 'Fake Thunder',
+        fr: 'Fausse foudre',
+        de: 'Falscher Blitz',
+      },
+      tts: {
+        en: 'Fake',
+        fr: 'Fausse',
+        de: 'Falsch',
+      },
     },
     {
       id: 'O8N Blizzard Fake Donut',
       regex: /14:2916:Kefka starts using Blizzard Blitz/,
+      regexDe: /14:2916:Kefka starts using Erstarrendes Eisga/,
+      regexFr: /14:2916:Kefka starts using Méga Glace Glissante/,
       suppressSeconds: 1,
-      infoText: 'Fake: Get Out',
-      tts: 'fake',
+      infoText: {
+        en: 'Fake Ice: Get out',
+        fr: 'Fausse glace: Sortir',
+        de: 'Falsches Eis: Rausgehen',
+      },
+      tts: {
+        en: 'Get out',
+        fr: 'Sortir',
+        de: 'Rausgehen',
+      },
     },
     {
       id: 'O8N Blizzard True Donut',
       regex: /14:2919:Kefka starts using Blizzard Blitz/,
+      regexDe: /14:2919:Kefka starts using Erstarrendes Eisga/,
+      regexFr: /14:2919:Kefka starts using Méga Glace Glissante/,
       suppressSeconds: 1,
       infoText: 'True: Get In',
       tts: 'true',
     },
     {
       id: 'O8N Blizzard Fake Near',
       regex: /14:2914:Kefka starts using Blizzard Blitz/,
+      regexDe: /14:2914:Kefka starts using Erstarrendes Eisga/,
+      regexFr: /14:2914:Kefka starts using Méga Glace Glissante/,
       suppressSeconds: 1,
-      infoText: 'Fake: Get In',
-      tts: 'fake',
+      infoText: {
+        en: 'Fake Ice: Get in',
+        fr: 'Fausse glace: Rentrer dedans',
+        de: 'Falsches Eis: Reingehen',
+      },
+      tts: {
+        en: 'Get in',
+        fr: 'rentrer dedans',
+        de: 'Reingehen',
+      },
     },
     {
       id: 'O8N Blizzard True Near',
       regex: /14:2918:Kefka starts using Blizzard Blitz/,
+      regexDe: /14:2918:Kefka starts using Erstarrendes Eisga/,
+      regexFr: /14:2918:Kefka starts using Méga Glace Glissante/,
       suppressSeconds: 1,
-      infoText: 'True: Get Out',
-      tts: 'true',
+      infoText: {
+        en: 'True Ice: Get out',
+        fr: 'Vraie glace: Sortir',
+        de: 'Wahres Eis: Rausgehen',
+      },
+      tts: {
+        en: 'Get out',
+        fr: 'Sortir',
+        de: 'Rausgehen',
+      },
+    },
+  ],
+  timelineReplace: [
```

Comment:
I think it's fine to just copy and paste this here.  It seems more work than it's worth to separate out what's included and what's not without an automatic tool to do it.

Reply:
Yup, I think so, too. Though when it's done you might have to take a look at the function closing brackets if I didn't use one too much or one is missing, since I don't see, which bracket closes what that good. I'll copy it into notepad++ though. Maybe it shows which bracket belongs to which and if there's some overflow of brackets x)